src/model/error.ml

```diff
@@ -107,3 +107,26 @@ let shed_warnings with_warnings =
   with_warnings.value
 
 let set_warn_error b = warn_error := b
+
+let implicit_acc = ref []
```

Comment:
This is yet an other way to accumulate warnings using a global mutable variable.